spec/haskell/src/SEL4/Object/ObjectType/ARM.lhs

```diff
@@ -218,12 +218,20 @@ Create an architecture-specific object.
 >             modify (\ks -> ks { gsUserPages =
 >               funupd (gsUserPages ks)
 >                      (fromPPtr regionBase) (Just ARMSmallPage)})
+>             when (not isDevice) $ doMachineOp $
```

Comment:
nitpick: when (not P) == unless P

Reply:
True, but the `not` lines up more nicely with C